pages/signup.js

```diff
@@ -1,9 +1,9 @@
-import SignUpForm from '../components/SignUpForm';
+import SignUpForm from '../containers/SignUpForm';
```

Comment:
Components dumb, containers smart
If component is using data from apollo or redux - its container
SignUpForm isnt component, its container
All components here is AuthFields and LinkList

**lets war begin!!!**